R/init.R

```diff
@@ -12,13 +12,12 @@
 #' @export
 vt_use_validation <- function(pkg = ".") {
 
+  validation_directory <- getOption("vt.validation_directory",default = "vignettes/validation")
```

Comment:
can we have $get and $set access for future extensibility? similar to `knitr::opts_knit`

Reply:
absolutely. I was getting a little concerned with the scope of the PR, so was waiting on that.

Reply:
yeah, let's just open a new issue for this

Reply:
see #19